packages/common/src/extensions/slickCellExternalCopyManager.ts

```diff
@@ -345,6 +371,12 @@ export class SlickCellExternalCopyManager {
     }
   }
 
+  protected trigger<ArgType = any>(evt: SlickEvent, args?: ArgType, e?: Event | SlickEventData) {
```

Comment:
so this looks like a full copy of the `trigger` method from SlickGrid, so perhaps we could just make the SlickGrid method as a public method and call it from here (and probably better to rename it as something like `triggerEvent`, or any other name, to be a little more specific)

Reply:
I was hesitant to ask for that but yep this would be my prefered solution as well